projects/ngx-charts-on-fhir/src/lib/data-layer-list/annotation-list/annotation-list.component.html

```diff
@@ -0,0 +1,23 @@
+<h3>Annotation Options</h3>
+
+<div class="scrolling-list">
+  <mat-accordion displayMode="flat" multi="true">
+    <mat-expansion-panel *ngFor="let annotation of _annotations; trackBy: trackByIndex" expanded="true">
+      <mat-expansion-panel-header>
+        <mat-panel-title>
+          <mat-checkbox
+            (click)="$event.stopPropagation()"
+            (change)="onCheckboxChange(annotation, $event)"
+            [checked]="annotation.display"
+            [title]="annotation.display ? 'Show this annotation on the chart' : 'Hide this annotation on the chart'"
+            color="primary"
+            id="annotationDisplayCheckBox"
+          ></mat-checkbox>
+          {{ annotation.label.content }}
+        </mat-panel-title>
+        <mat-panel-description> Annotation </mat-panel-description>
+      </mat-expansion-panel-header>
+      <annotation-options [annotation]="annotation" (onAnnotationsChange)="this.onAnnotationsChange(annotation, $event)"></annotation-options>
```

Comment:
forgot to update this when changing event name
```suggestion
      <annotation-options [annotation]="annotation" (annotationChange)="this.onAnnotationsChange(annotation, $event)"></annotation-options>
```